README.md

```diff
@@ -131,8 +129,6 @@ The live image boots to UEFI Shell. The different test applications can be run i
 - [UEFI Self Certification Tests (UEFI-SCT)](https://github.com/UEFI/UEFI-SCT) TAG: c78ea66cb114390e8dd8de922bdf4ff3e9770f8c
```

Comment:
Shouldn't be `https://github.com/UEFI/UEFI-SCT`  changed to `https://github.com/tianocore/edk2-test`